locales/translations/fr.json

```
@@ -58,6 +58,8 @@
     "activities": "activités",
     "activity": "activité",
     "Activity": "Activité",
+    "Actual success rate": "Taux de réussite réel",
+    "Actual success rate value throughout the survey's life": "Valeur réelle du taux de réussite tout au long de la durée de vie de l'enquête",
```

Hey, IA translators ain't bad :+1:
I'd just simplify this sentence a little bit:

```suggestion
    "Actual success rate value throughout the survey's life": "Taux de réussite réel sur toute la durée de l'enquête",
```